sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -143,6 +145,9 @@ private[sql] class SQLConf extends Serializable with CatalystConf {
   private[spark] def parquetUseDataSourceApi =
     getConf(PARQUET_USE_DATA_SOURCE_API, "true").toBoolean
 
+  private[spark] def orcFilterPushDown =
+    getConf(ORC_FILTER_PUSHDOWN_ENABLED, "false").toBoolean
```

Comment:
How flaky is this? I'd maybe turn it on by default during QA, and then consider disabling it when we release if it is indeed flaky.